managedtenants/utils/ocm.py

```diff
@@ -50,6 +50,8 @@ class OcmCli:
         "subOperators": "sub_operators",
         "managedService": "managed_service",
         "config": "config",
+        "namespaces": "namespaces",
+        "namespaceLabels": "namespaceLabels",
```

Comment:
`namespaceLabels` is not required here as we don't send that to ocm. Otherwise looks good

Reply:
cool. removing it. 🙂